spec/features/projects_manage_spec.rb

```diff
@@ -35,6 +35,10 @@
     it "allow me to clone the project" do
       expect(page).to have_selector(:link_or_button, "Clone Project", disabled: false)
     end
+
+    it "has an archived label" do
+      expect(page).to have_selector("span.archived")
```

Comment:
Let's use the have_text assertion rather if possible.
In essence, we don't need to test the classes if we have confidence that the right text is shown.
`expect(page).to have_text("Archived", count: 1)`